src/main/java/com/task/amazon/entities/AmazonEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.task.amazon.entities;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "entities")
+public class AmazonEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String productId;
```

Comment:
@karsv could we use this as Id? Or the 'productId' field is not unique?

Reply:
We can use id's from csv file.